src/rebar_git_resource.erl

```diff
@@ -335,7 +335,8 @@ parse_tags(Dir) ->
         {ok, Line} ->
             case re:run(Line, "(\\(|\\s)(HEAD[^,]*,\\s)tag:\\s(v?([^,\\)]+))", [{capture, [3, 4], list}, unicode]) of
                 {match,[Tag, Vsn]} ->
-                    {Tag, Vsn};
+                    %% git tag can be repo_name/proj/05323498<hash>38, only want the hash
+                    {Tag, lists:last(string:lexemes(Vsn, "/"))};
```

Comment:
`string:lexemes` isn't a safe function to call in rebar3 for older support; please call [rebar_string](https://github.com/erlang/rebar3/blob/master/src/rebar_string.erl):lexemes/2 instead, which wraps the API for older versions.